bin/run.sh

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env bash
+
+set -e
+
+unset CDPATH
+cd "$( dirname "${BASH_SOURCE[0]}" )/.."
+
+echo() { builtin echo "[SCITRAN] $@"; }
+
+
+set -o allexport
+
+if [ "$#" -eq 1 ]; then
+    EXISTING_ENV=$(env | grep "SCITRAN_" | cat)
+    source "$1"
+    eval "$EXISTING_ENV"
+fi
```

Comment:
In that case, I'd leave it like this. I'm not a huge fan of trying to coalesce environment sets, or saving config in that format, but if we're going with that this method LGTM.